### DISCUSSION TITLE
Internet problem after last update

### DISCUSSION THREAD

**Discussion by mohammedxhameed** [pivpn/pivpn#1536] [Support]
Hello 
I have a problem, which is weak internet for customers after the last update, what is the reason?

**Comment by coolapso** [comment]
When you go to the doctor you also only say, "Hello mister doctor, It hurts, can you fix it?" 
You need to be more specific. 

which Update? PiVPN is not a software that you update, sounds to me that you have been messing with the installation now its broken. 

When you created "new issue" you were presented with a template that you should have filled in. Please get the template here: https://raw.githubusercontent.com/pivpn/pivpn/master/.github/ISSUE_TEMPLATE/pivpn-report.md

Edit your original post and edit it with the details requested in that template. 

I have converted this into a discussion.

**Comment by mohammedxhameed** [comment]
When I connect through the vpn, the connection is very bad, knowing that this problem has gradually occurred with the latest version of pivpn

As you can see in the picture, I used 11 GB of internet, but it only gave me a very bad connection

![image](https://user-images.githubusercontent.com/55911642/170846812-e7af90a0-3e06-4e95-9eab-9e131331143d.png)
